provider/aci_test.go

```diff
@@ -1191,3 +1191,151 @@ func TestCreatePodWithProjectedVolume(t *testing.T) {
 		t.Fatal("Failed to create pod", err)
 	}
 }
+
+func TestCreatePodWithCSIVolume(t *testing.T) {
+	podName := "pod-" + uuid.New().String()
+	podNamespace := "ns-" + uuid.New().String()
```

Comment:
Why use a random string here? Do they have to be different?

Reply:
Same. Followed the code convention. However, I'm ok with them being random.